go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/cenkalti/backoff/v4 v4.2.0
 	github.com/charmbracelet/glamour v0.5.1-0.20220727184942-e70ff2d969da
 	github.com/charmbracelet/lipgloss v0.5.0
-	github.com/cli/go-gh v1.2.1
+	github.com/cli/go-gh v1.2.2-0.20230409230332-f35ae2c6171b
```

Comment:
Once the go-gh PR is merged and v2 is actually released I will update the PR to use the version number instead of the SHA.